src/vm/docs/mod.rs

```diff
@@ -173,6 +173,54 @@ const TO_INT_API: SimpleFunctionAPI = SimpleFunctionAPI {
     example: "(to-int u238) ;; Returns 238"
 };
 
+const BUFF_TO_INT_LE_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(buff-to-int-le (buff 16))",
+    description: "Converts a byte buffer to a signed integer use a little-endian encoding.",
```

Comment:
Can you add a description of the padding behavior to the descriptions of these functions?

Reply:
done